project/app/api/models.py

```diff
@@ -148,3 +146,49 @@ def check_valid_subid(cls, value):
     def check_sha_len(cls, value):
         assert len(value) == 128
         return value
+
+
+class LineGraphRequest(BaseModel):
+    """Model that handles LineGraph Request to API
+    """
+
+    ScoreHistory: List[int] = Field(..., example=[1005, 1500, 9000, 789])
+    StudentName: str = Field(..., example="Firstname")
+
+
+class HistogramRequest(BaseModel):
+    """Model that handles Histogram Request to API
+    """
+
+    GradeList: List[int] = Field(..., example=[1005, 1500, 9000, 789])
+    GradeLevel: int = Field(..., example=8)
+    StudentName: str = Field(..., example="Firstname")
+    StudentScore: int = Field(..., example=1058)
+
+
+class ClusterSubmission(BaseModel):
+    Image: str = Field(..., example="http://lorempixel.com/640/480/abstract")
```

Comment:
Can you add a docstring to this class and the class below, just for consistency?